### DISCUSSION TITLE
30s delay in tray icons showing when Win11 startmenu enabled.

### DISCUSSION THREAD

**Discussion by b1k3rdude** [valinet/ExplorerPatcher#2012] [Q&A]
So it took me a while of clicking things untill I accidentially came across the work around for the issue I was having.

Under the startmenu section> start menu style:  - if the 'default Windows 11' option is enabled and you click 'Restart File Explorer' there is a 30 second delay before any of the tray icons will appear.  

However when rebooting windows, the icons then dont appear at all. So the second part of the work around is you have to set the start menu button to Win10 also otherwise all the non-system tray icons will not show.

I have tested the above my Win11 virtual machine (22000.2360, VmwarePlayer) and my Media PC live test machine (22000.2295).

**Comment by pyrates999** [comment]
The win11 vm has a preview update installed: https://support.microsoft.com/en-us/topic/august-22-2023-kb5029332-os-build-22000-2360-preview-8f8aec64-77b4-4225-9a0f-f0153204ae28

Uninstall kb5029332.

You're also on 21h2.  Please update to 22h2.

**Comment by b1k3rdude** [comment]
So your saying the current version of ExplorerPatcher is only compliant with 22H2 and not 21H2..?

**Reply by pyrates999** [reply]
I would update to 22h2.  As a general rule during troubleshooting of software, make sure you're on the latest release of the software.
